client/src/pages/Register.jsx

```diff
@@ -99,14 +144,14 @@ const Register = ({ form }: Props) => {
             rules: [
               {
                 required: true,
-                message: 'Please create a password!',
+                message: intl.formatMessage(messages.pleaseCreateAPassword),
               },
             ],
           })(
             <Input.Password
               prefix={<LockOutlined style={{ color: 'rgba(0,0,0,.25)' }} />}
               type="password"
-              placeholder="Create Password"
+              placeholder={intl.formatMessage(messages.pleaseCreateAPassword)}
```

Comment:
This might have been a copy/paste error - this should probably be `messages.createPassword`